src/main/resources/config/application.properties.sample

```diff
@@ -14,24 +14,34 @@ spring.thymeleaf.mode=HTML5
 spring.thymeleaf.encoding=UTF-8
 spring.thymeleaf.content-type=text/html
 
-##DATABASE PROPERTIES
+## DATABASE PROPERTIES
 spring.datasource.driverClassName=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://localhost:5432/<MyDB>
-spring.datasource.username=<dbUser>
-spring.datasource.password=<dbUserPassword>
+# Change <MyDB> with the name of your database name
+#spring.datasource.url=jdbc:postgresql://localhost:5432/<MyDB>
+# Username to log in to the database
```

Comment:
Perhaps add "It's recommended to create a dedicated AreaFiftyLAN Database user instead of using the default postgres user"